css/style.css

```diff
@@ -0,0 +1,234 @@
+body {
+  background-color: #eee;
+  display: flex;
+  flex-direction: column;
+  font-family: sans-serif;
+  color: #000;
+}
+
+header {
+  position: fixed;
+  width: 100%;
+  background-color: #eee;
+  display: grid;
+  grid-template-columns: 300px 1fr;
+  z-index: 1;
+  top: 0;
+  left: 0;
+  padding-top: 10px;
+}
+
+main {
+  display: flex;
+  flex-direction: column;
+  margin: auto;
+  max-width: 750px;
+  position: relative;
+  top: 40px;
+  z-index: 0;
+}
+
+#header-img {
+  grid-column: 1;
+  width: 100%;
+  align-self: center;
+}
+
+#nav-bar {
+  grid-column: 2;
+  align-self: center;
+  justify-self: right;
+  display: flex;
+  margin-right: 5px;
+}
+
+.nav-link {
+  text-decoration: inherit;
+  color: inherit;
+  display: inline-block;
+  margin: 5px;
+}
+
+/* input0[type=submit] {
+      text-align: center;
+  } */
```

Comment:
Kindly, remove **unused code as a comment in your codebase** as It will make your codebase cleaner and easy to follow.